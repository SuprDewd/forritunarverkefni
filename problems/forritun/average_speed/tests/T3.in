00:06:21
00:13:40 102
00:17:29
00:19:13
00:24:26 21
00:29:19 186
00:35:27
00:42:05 142
00:47:44 57
00:48:49 94
00:49:55
00:58:05
01:06:18 134
01:10:28
01:16:44
01:21:56 21
01:24:33 74
01:32:13 70
01:39:48 179
01:47:35 54
01:49:02
01:54:28
01:54:47
02:00:56 174
02:01:48 51
02:02:54 40
02:10:44
02:12:47
02:18:14 47
02:18:30 131
02:21:42
02:25:22
02:30:46
02:35:54 28
02:42:39
02:45:27
02:47:20
02:51:51
02:58:42 87
02:59:10
03:03:03 90
03:10:15
03:14:21 164
03:16:14
03:18:54 200
03:24:18 88
03:29:29 140
03:36:57
03:38:23 94
03:41:28
03:43:46
03:48:50 107
03:51:51 179
03:58:37
03:59:46
04:02:52
04:08:49 167
04:15:47 192
04:21:25 95
04:22:49
04:26:39
04:31:52
04:37:33
04:37:34 47
04:41:38 169
04:49:39
04:54:32 124
04:58:47 143
04:59:13
05:02:38 34
05:03:53 162
05:03:58
05:08:27
05:14:43
05:16:45
05:16:50 174
05:22:40
05:29:23 192
05:29:28 34
05:31:36 25
05:32:30 196
05:34:57
05:40:21 139
05:44:33 117
05:46:14
05:48:59
05:52:24 40
05:52:57 71
06:00:03
06:06:30 40
06:11:52
06:19:11
06:27:07
06:34:08 42
06:37:21 98
06:40:37 176
06:41:26
06:47:50
06:53:07 121
07:00:50
07:01:56 164
07:08:16
07:15:34 152
07:18:00
07:22:54 140
07:28:02
07:31:01
07:35:02
07:41:47 115
07:49:59
07:52:52 70
07:58:37 37
08:04:27 106
08:07:26 19
08:13:55 114
08:20:03 159
08:22:51 65
08:24:28
08:28:23
08:30:08
08:31:38 180
08:35:19 50
08:39:54 147
08:41:38
08:44:30 17
08:49:35 176
08:52:40 191
08:56:56 47
09:05:12
09:08:34
09:14:31 194
09:19:31 152
09:22:07 192
09:28:32 74
09:30:38 133
09:32:43 15
09:37:24
09:43:47 186
09:49:09 132
09:50:02
09:51:23
09:52:15
09:55:36
10:02:53 37
10:08:14 56
10:10:43
10:15:58 67
10:21:40 83
10:23:06
10:23:11 12
10:31:30 107
10:36:38 3
10:37:05 7
10:43:06
10:43:37
10:44:23
10:49:46
10:55:40
10:59:17
10:59:38 152
11:07:31
11:14:25 199
11:19:40
11:20:15 172
11:25:45 61
11:33:30
11:35:32
11:40:44 62
11:48:19 160
11:55:16 107
12:02:04
12:02:50
12:10:58 8
12:12:35 13
12:13:45
12:20:16
12:26:51
12:27:35 85
12:31:21
12:35:02
12:35:57
12:42:26
12:43:38 157
12:48:39 30
12:51:36
12:54:13 96
12:56:53 198
13:05:02 168
13:08:08 88
13:14:17
13:19:40
13:24:13 123
13:27:39
13:34:51
13:42:35
13:44:16
13:45:29
13:51:26
13:58:51
14:04:25 78
14:08:29 85
14:13:27 45
14:17:36
14:24:25 144
14:31:23 154
14:37:23 183
14:44:28
14:49:56 12
14:53:14 60
15:00:48
15:03:28 85
15:09:27 194
15:12:14 136
15:12:59 167
15:21:00
15:26:48
15:31:13
15:33:02
15:41:06
15:47:47 75
15:54:03 57
16:02:22
16:10:04
16:15:07
16:19:09
16:24:40 77
16:27:56
16:33:17
16:34:34 3
16:39:38
16:45:49
16:49:55
16:53:34
17:01:37
17:06:07 89
17:07:45
17:11:23
17:14:14
17:17:59 84
17:22:39 73
17:23:24
17:29:54 5
17:34:01
17:40:35 165
17:47:06
17:47:19 180
17:49:44 80
17:50:37
17:55:26
17:58:06 186
18:00:18
18:04:20
18:09:25 104
18:15:24
18:22:51
18:26:19 157
18:28:39
18:31:49
18:34:33 78
18:37:45
18:42:28
18:48:02
18:51:40
18:56:56 153
18:59:48
19:07:40
19:13:55 50
19:17:36 103
19:21:15 91
19:22:14 185
19:26:55 133
19:30:52 113
19:37:25 126
19:42:05
19:44:27 42
19:46:27 165
19:47:20 113
19:52:08 25
19:56:48
19:59:52
20:02:40 121
20:09:34 125
20:10:32 184
20:15:15
20:22:32 184
20:30:18
20:33:49 28
20:38:15
20:39:50
20:41:57 135
20:46:25 129
20:54:19 126
21:01:33 5
21:03:48
21:03:57 52
21:06:08 149
21:10:36 185
