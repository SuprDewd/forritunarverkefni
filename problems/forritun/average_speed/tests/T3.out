00:06:21 0.00 km
00:17:29 6.49 km
00:19:13 9.44 km
00:35:27 39.03 km
00:49:55 75.71 km
00:58:05 88.51 km
01:10:28 110.69 km
01:16:44 124.68 km
01:49:02 180.04 km
01:54:28 184.93 km
01:54:47 185.21 km
02:10:44 199.42 km
02:12:47 200.79 km
02:21:42 211.61 km
02:25:22 219.62 km
02:30:46 231.41 km
02:42:39 245.77 km
02:45:27 247.07 km
02:47:20 247.95 km
02:51:51 250.06 km
02:59:10 253.94 km
03:10:15 270.37 km
03:16:14 281.66 km
03:36:57 331.98 km
03:41:28 340.15 km
03:43:46 343.76 km
03:58:37 377.26 km
03:59:46 380.69 km
04:02:52 389.94 km
04:22:49 447.32 km
04:26:39 453.39 km
04:31:52 461.65 km
04:37:33 470.65 km
04:49:39 496.44 km
04:59:13 520.01 km
05:03:58 529.09 km
05:08:27 541.20 km
05:14:43 558.12 km
05:16:45 563.61 km
05:22:40 580.75 km
05:34:57 610.08 km
05:46:14 640.73 km
05:48:59 646.09 km
06:00:03 661.53 km
06:11:52 672.74 km
06:19:11 677.61 km
06:27:07 682.90 km
06:41:26 697.56 km
06:47:50 716.34 km
07:00:50 747.40 km
07:08:16 766.93 km
07:18:00 793.04 km
07:28:02 817.43 km
07:31:01 824.40 km
07:35:02 833.77 km
07:49:59 865.23 km
08:24:28 909.21 km
08:28:23 913.46 km
08:30:08 915.35 km
08:41:38 936.09 km
09:05:12 973.66 km
09:08:34 976.30 km
09:37:24 1032.62 km
09:50:02 1052.80 km
09:51:23 1055.77 km
09:52:15 1057.68 km
09:55:36 1065.05 km
10:10:43 1086.69 km
10:23:06 1099.94 km
10:43:06 1111.59 km
10:43:37 1111.65 km
10:44:23 1111.74 km
10:49:46 1112.37 km
10:55:40 1113.06 km
10:59:17 1113.48 km
11:07:31 1133.49 km
11:19:40 1168.39 km
11:33:30 1193.97 km
11:35:32 1196.03 km
12:02:04 1239.82 km
12:02:50 1241.18 km
12:13:45 1256.16 km
12:20:16 1257.57 km
12:26:51 1258.99 km
12:31:21 1264.49 km
12:35:02 1269.71 km
12:35:57 1271.01 km
12:42:26 1280.19 km
12:51:36 1296.49 km
13:14:17 1346.66 km
13:19:40 1354.56 km
13:27:39 1368.27 km
13:34:51 1383.03 km
13:42:35 1398.88 km
13:44:16 1402.33 km
13:45:29 1404.83 km
13:51:26 1417.03 km
13:58:51 1432.23 km
14:17:36 1459.08 km
14:44:28 1517.91 km
15:00:48 1542.81 km
15:21:00 1586.97 km
15:26:48 1603.11 km
15:31:13 1615.41 km
15:33:02 1620.46 km
15:41:06 1642.91 km
16:02:22 1677.25 km
16:10:04 1684.56 km
16:15:07 1689.36 km
16:19:09 1693.19 km
16:27:56 1702.63 km
16:33:17 1709.49 km
16:39:38 1711.39 km
16:45:49 1711.70 km
16:49:55 1711.91 km
16:53:34 1712.09 km
17:01:37 1712.49 km
17:07:45 1715.14 km
17:11:23 1720.53 km
17:14:14 1724.76 km
17:23:24 1737.77 km
17:34:01 1746.02 km
17:47:06 1764.48 km
17:50:37 1773.51 km
17:55:26 1779.93 km
18:00:18 1790.31 km
18:04:20 1802.81 km
18:15:24 1828.94 km
18:22:51 1841.85 km
18:28:39 1853.97 km
18:31:49 1862.25 km
18:37:45 1873.57 km
18:42:28 1879.70 km
18:48:02 1886.93 km
18:51:40 1891.66 km
18:59:48 1905.81 km
19:07:40 1925.87 km
19:42:05 1997.97 km
19:56:48 2017.75 km
19:59:52 2019.03 km
20:15:15 2050.59 km
20:30:18 2096.74 km
20:38:15 2109.60 km
20:39:50 2110.34 km
21:03:48 2153.74 km
